### PR TITLE
Add special layout for Solari for screen cards

### PR DESCRIPTION
**Pull request by hannahpurcell**
[Notion](https://www.notion.so/mbta-downtown-crossing/Solari-screens-should-stack-side-by-side-072a894f2d8440a4ae262f6b1fb04e4c)

If the screen type is Solari, we want to wrap all said Solari screens in a different layout component that lets the screen cards flex and fill a row. (Currently, we only ever expect to see <2 Solari screens, and the flex method looks good up to 3 screens.)

### CODE REVIEW

File: assets/js/components/Dashboard/ScreenDetail.tsx [hunk 1]
```diff
@@ -11,18 +11,30 @@ interface ScreenDetailProps {
...
-  };
+const ScreenCard = (props: ScreenDetailProps) => {
+  const { screens, isOpen } = props;
+  const isPaess = screens.every((screen) => screen.type === "pa_ess");
+  const isSolari = screens.every((screen) => screen.type === "solari");
+  const paessRouteLetter = screens[0].station_code
+    ? screens[0].station_code.charAt(0).toLowerCase()
+    : "";
+
+  const translateScreenType = SCREEN_TYPES.find(({ ids }) =>
```
Comment (cmaddox5) [inline assets/js/components/Dashboard/ScreenDetail.tsx:35]: Super nitpicky, but the name of this still reads like it is a function. What do you think about `translatedScreenType`?

**Review by cmaddox5 — APPROVED** [review]
Looks great! Just have one optional suggestion.

File: assets/js/components/Dashboard/ScreenDetail.tsx (hunk 1 above)
Comment (hannahpurcell) [reply assets/js/components/Dashboard/ScreenDetail.tsx:35]: oh yeah, definitely worth mentioning! thanks

Comment (hannahpurcell) [inline assets/js/components/Dashboard/ScreenDetail.tsx:35]:
```suggestion
  const translatedScreenType = SCREEN_TYPES.find(({ ids }) =>
```

**Pull request merged**